packages/components/checkbox/src/uses.ts

```diff
@@ -0,0 +1,121 @@
+import { computed, ComputedRef, inject, Ref, ref, watch, getCurrentInstance } from 'vue'
+import { checkboxGroupInjectionKey, emitterInjectKey } from './checkbox'
+import { isArray } from '@idux/cdk/utils'
+
+type CheckValue = string | number | boolean
+
+interface UseCheckValue {
+  (isDisable: ComputedRef<boolean>): { checkeValue: Ref<CheckValue>; handleChange: (e: Event) => void }
+}
+
+interface UseChecked {
+  (checkValue: Ref<CheckValue>): ComputedRef<boolean>
+}
+
+interface UseDisabled {
+  (): ComputedRef<boolean>
+}
+
+interface UseClasses {
+  (isDisabled: ComputedRef<boolean>, isChecked: ComputedRef<boolean>): ComputedRef<
+    [
+      {
+        'ix-checkbox-disabled': boolean
+        'ix-checkbox-indeterminate': boolean
+        'ix-checkbox-checked': boolean
+      },
+    ]
+  >
+}
+
+interface UseName {
+  (): ComputedRef<string>
+}
+
+const useCheckValue: UseCheckValue = isDisabled => {
```

Comment:
```suggestion
export const useCheckValue = (isDisabled: Ref<boolean>): { checkeValue: Ref<CheckValue>; handleChange: (e: Event) => void }
=> {
```

或者把 `{ checkeValue: Ref<CheckValue>; handleChange: (e: Event) => void }` 定义成一个类型